### PR TITLE
Initial project setup backend

### PR DESCRIPTION
**Pull request by Solodkih**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>TT Sunday School</title>
```
Comment (porlov) [inline frontend/public/index.html]: Please use project name)

File: frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <link rel="shortcut icon" href="/static/favicon.ico" />
+    <link rel="stylesheet" href="/static/dist/styles.css">
+  </head>
+  <body>
+    <h2 class="first2">Hello world</h2>
+    <div id="app" />
+  </body>
+</html>
```
Comment (porlov) [inline frontend/public/index.html]: empty line

File: backend/backend/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+__pycache__/
```
Comment (porlov) [inline backend/backend/.gitignore]: Why do we need to have `backend` in `backend` folder?

File: backend/backend/settings.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+"""
+
+import os, sys
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+PROJECT_ROOT = os.path.dirname(__file__)
+#sys.path.insert(0, os.path.join(PROJECT_ROOT, '../../frontend/public))
```
Comment (porlov) [inline backend/backend/settings.py]: Please remove

**Comment by porlov** [comment]
General note: Why do we need `.keep` files?

File: frontend/public/index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```
Comment (porlov) [inline frontend/public/index.html]: Please do all frontend changes in frontend PR

**Comment by porlov** [comment]
General note: we should have readme file that describes how to run the application (i think it should be one file for both backend and frontend part)

File: backend/backend/urls.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+"""
+from django.conf.urls import url
+from django.contrib import admin
+from . import views 
+from django.urls import path
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    path('', views.index, name='index')
+]
```
[the remark this answers is not in the document]
Comment (Solodkih) [reply backend/backend/urls.py]: What did you mean? Should the empty string be 3 arguments?

File: frontend/public/index.html (hunk 1 above)
Comment (Solodkih) [reply frontend/public/index.html]: I am change it in init-setup-frontend

File: frontend/public/index.html (hunk 2 above)
Comment (Solodkih) [reply frontend/public/index.html]: I am change it in init-setup-frontend

File: backend/backend/.gitignore (hunk 3 above)
Comment (porlov) [reply backend/backend/.gitignore]: painters_gallery

**Pull request merged**